src/@next/Icon/icons/ri-custom-salary-fill.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 25" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@music1353 can you check both icons? it seems there's a bit of a difference with the size from the export. we usually get it at 24 x 24

Reply:
Sure, I can adjust to 24 x 24